### PR TITLE
Add Bouygue Flexom server definition

### PR DESCRIPTION
**Pull request by phimage**
Hi,
Just adding another server endpoint
Flexom v3 is another rebranding of overkiz cloud, here for Bouygue

I could list my devices using my login/password and code from README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +22,7 @@ This package is written for the Home Assistant [ha-tahoma](https://github.com/iM
 - Somfy TaHoma
 - Somfy TaHoma Switch
 - Thermor Cozytouch
+- Bouygue Flexom
```
Comment (tetienne) [inline README.md:25]:
```suggestion
- Bouygues Flexom
```

File: pyoverkiz/const.py [hunk 2]
```diff
@@ -71,4 +71,10 @@
         manufacturer="Somfy",
         configuration_url=None,
     ),
+    "flexom": OverkizServer(
+        name="Flexom",
+        endpoint="https://ha108-1.overkiz.com/enduser-mobile-web/enduserAPI/",
+        manufacturer="Bouygue",
```
Comment (tetienne) [inline pyoverkiz/const.py:77]:
```suggestion
        manufacturer="Bouygues",
```

**Review by tetienne — CHANGES_REQUESTED** [review]
Some typo.

Thx for your contributions. I didn't know this solution. https://www.overkiz.com/flexom-3-0-le-logement-connecte-par-bouygues-immobilier-et-overkiz/

Have you been able to call the `get_setup` method?

**Comment by phimage** [comment]
I will fix the typo

I launch `get_setup` and it failed to decode my gateway
Value 47 for HATTARA RAIL DIN https://www.overkiz.com/wp-content/uploads/2018/10/FICHE_PRODUIT_DIN_RAIL_2019_FR-.pdf
I will add it in the PR
 
I see that some location info are empty like country
(in another library that use this info to load some translation file it make it crash 😣)

**Review by iMicknl — APPROVED** [review]
Good find! Do you plan to use the HA integration?

**Comment by iMicknl** [comment]
@phimage could you run the `get_diagnostic_data()` function perhaps? It would be good to add a setup.json definition for your device as well, for our automated tests.

**Comment by phimage** [comment]
> Good find! Do you plan to use the HA integration?

I will make a try, but not now, it’s a new appartement and I do not live in it yet

I have just tested HomeBrige for HomeKit, it’s laggy

> @phimage could you run the `get_diagnostic_data()` function perhaps? It would be good to add a setup.json definition for your device as well, for our automated tests.

I will

**Comment by mbrd1** [comment]
You mention Flexom V3 but there is also Flexom V2 which uses the same hardware but uses the Ubiant Cloud.
Do you think something is possible for this one too?

3222D6CB-5D3A-48D8-B66A-BABDE73C3340

**Comment by iMicknl** [comment]
@mbrd1 how do you connect to Flexom V2? If you can link me the Android app, I can have a look if it would be supported.

**Comment by mbrd1** [comment]
@iMicknl 

With an mail/password but it uses the Ubiant server : hemisphere.ubiant.com

I have contacted Overkiz to see if it is possible not to go through Ubiant, I am waiting for their answer.

Flexom V2 : 
https://play.google.com/store/apps/details?id=com.ubiant.flexom&hl=fr&gl=US

**Comment by phimage** [comment]
Hi,
for me v2 is different
https://github.com/rsauget/flexom-lib
with different url
https://github.com/rsauget/flexom-lib/blob/b92f332e319367651df6c10fd45a0e932da5f71f/src/ubiant/ubiant.ts#L19
hemis , like v2 app
https://play.google.com/store/apps/details?id=com.ubiant.flexom&hl=fr&gl=US

but maybe they make a bridge

**Pull request merged**